src/components/profile/profileNav/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { BottomNavigation, BottomNavigationAction } from '@mui/material';
+
+interface Props {
+  value: string | number;
+  navigationData: { label: number; title: string }[];
+  onChange:
+    | ((event: React.SyntheticEvent<Element, Event>, value: string) => void)
+    | undefined;
```

Comment:
이런 방식으로 optional 하게 해줘도 괜찮지 않을까요??
```suggestion
  onChange?: (event: React.SyntheticEvent<Element, Event>, value: string) => void;
```